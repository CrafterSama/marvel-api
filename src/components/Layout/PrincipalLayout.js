import React from 'react'
import Header from '../Common/Header'
import Footer from '../Common/Footer'
import Home from '../Home'
import {
  Layout
} from '../../styles'
import { useStateValue } from '../../context'

const PrincipalLayout = (props) => {
  const [state] = useStateValue()
  return (
    <Layout darkmode={state.darkMode}>
      <Header />
      <Home />
      <Footer />
    </Layout>
  )
}

export default PrincipalLayout
