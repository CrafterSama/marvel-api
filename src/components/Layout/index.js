import React from 'react'
import Header from '../Common/Header'
import Footer from '../Common/Footer'
import Home from '../Home'
import {
  MainLayout
} from '../../styles'
import { useStateValue } from '../../context'

const Layout = (props) => {
  const [state] = useStateValue()
  return (
    <MainLayout darkmode={state.darkMode}>
      <Header />
      <Home />
      <Footer />
    </MainLayout>
  )
}

export default Layout
