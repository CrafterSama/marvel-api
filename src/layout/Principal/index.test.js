import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';

import Layout from './index';

test('matches snapshot', () => {
  const wrapper = shallow(<Layout />);

  expect(toJSON(wrapper)).toMatchSnapshot();
})
