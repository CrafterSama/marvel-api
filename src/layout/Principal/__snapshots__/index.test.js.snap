// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <Layout /> Component matches snapshot 1`] = `
<styles__MainLayout
  darkmode=""
>
  <Header />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      path="/comics/:id"
    />
  </Switch>
</styles__MainLayout>
`;
